internal/characters/sara/burst.go

```diff
@@ -80,8 +80,8 @@ func (c *char) Burst(p map[string]int) action.ActionInfo {
 		direction := combat.DegreesToDirection(i * stepSize).Rotate(burstInitialDirection)
 		// 6 ticks per stormcluster
 		for j := 0; j < 6; j++ {
```

Comment:
I think this is 5 ticks and not 6 :/

The `AttackWave_DamageHandler` modifier has a duration of 1.4s and think interval of 0.3s. If `T` is the time when the think interval starts (`T = initial_hit_time + 0.8s`): 
1. tick at `T`
2. tick at `T + 0.3`
3. tick at `T + 0.6`
4. tick at `T + 0.9`
5. tick at `T + 1.2`

There is no 6th tick since it would be at `T + 1.5` which is after the modifier expires/gets removed (removed at `T + 1.4`).

5 ticks also matches the information in the [KQM TCL](https://library.keqingmains.com/evidence/characters/electro/kujou-sara#sara-burst-mechanics-update):
![image](https://user-images.githubusercontent.com/2059811/216243866-1aca8fff-147f-4f7b-9d62-80bb77a3c22e.png)